src/schema/v2/homeView/zones/default.ts

```diff
@@ -31,16 +31,16 @@ const SECTIONS: HomeViewSection[] = [
   LatestActivity,
   NewWorksForYou,
   DiscoverSomethingNew,
-  HeroUnits,
+  CuratorsPicksEmerging,
   ExploreByCategory,
+  HeroUnits,
   ActiveBids,
   AuctionLotsForYou,
   Auctions,
   LatestAuctionResults,
   GalleriesNearYou,
   LatestArticles,
   News,
-  CuratorsPicksEmerging,
   MarketingCollections,
```

Comment:
**question (non-blocking):** Just noticing the [comment here](https://artsy.slack.com/archives/C07ANEV7RNV/p1729860898364819) and wondering if `MarketingCollections` was meant to be removed here as well.